src/Temporalio/Common/Constants.cs

```diff
@@ -0,0 +1,13 @@
+namespace Temporalio.Common
+{
+    /// <summary>
+    /// Constants needed by the Temporal SDK.
+    /// </summary>
+    internal static class Constants
+    {
+        /// <summary>
+        /// Prefix for reserved handler and definition names.
+        /// </summary>
+        internal const string ReservedNamePrefix = "__temporal";
```

Comment:
_Technically_ `__temporal_`, but I am ok with this without trailing underscore

Reply:
I think we should probably actually make it specifically be without that. Seems very odd you could register `__temporalmything` but not `__temporal_mything`

Reply:
Yeah, was unsure here because in spec I added underscore not thinking and other SDKs already merged theirs, though I doubt an inconsistency here matters.